test/e2e/node/runtimeclass.go

```diff
@@ -35,7 +35,7 @@ import (
 	"github.com/onsi/gomega"
 )
 
-var _ = ginkgo.Describe("[sig-node] RuntimeClass", func() {
+var _ = SIGDescribe("RuntimeClass", func() {
```

Comment:
SIGNodeDescribe right?

Reply:
No - in test/e2e/node SIGDescribe already has [sig-node]

Reply:
this is a bit confusing, maybe worth keeping the name to SIGNodeDescribe